src/components/Comment/index.tsx

```diff
@@ -572,7 +571,7 @@ export default class CommentComponent extends React.Component<CommentProps> {
     };
 
     const onDoubleClick = () => {
-      this.props.store.dispatch(setPinnedComment(this.props.comment.localId));
+      this.props.store.dispatch(setFocusedComment(this.props.comment.localId, true));
```

Comment:
For extra boolean arguments, could we configure them so it can be called like `setFocusedComment(this.props.comment.localId, {pin: true})`?

Reply:
Done!